deeds-dapp-service/src/main/java/io/meeds/deeds/service/ExchangeService.java

```diff
@@ -47,52 +47,52 @@
 @Component
 public class ExchangeService {
 
-  private static final Logger            LOG                               = LoggerFactory.getLogger(ExchangeService.class);
```

Comment:
please apply formatting settings in your IDE https://developer.exoplatform.org/#id-ide-settings-to-work-on-exo-platform-projects and avoid formatting all class, only added code

Reply:
> and avoid formatting all class, only added code